snooty/main.py

```diff
@@ -299,10 +370,19 @@ def main() -> None:
         return
 
     url = args["<mongodb-url>"]
-    connection = (
-        None if not url else pymongo.MongoClient(url, password=getpass.getpass())
-    )
-    backend = MongoBackend(connection) if connection else Backend()
+    output_path = args["--output"]
+
+    connection: Optional[pymongo.MongoClient] = None
+
+    if url:
+        connection = pymongo.MongoClient(url, password=getpass.getpass())
+        backend: Backend = MongoBackend(connection)
+    elif output_path:
+        zf = zipfile.ZipFile(output_path, mode="w")
+        backend = ZipBackend(zf)
```

Comment:
Right -- I want to do away with the Atlas functionality entirely down the road, since I think having a pymongo dependency is a little weird for the parser.

Do you see a reason we'd want to use multiple backends in one build?

Reply:
Yup, makes sense!

I don't think it would be mandatory to have both, but there have been times where being able to see the AST in Atlas has been useful for debugging, especially for builds on the autobuilder. Straying away from this isn't the end of the world though since we can just run the parser locally to see output as needed

Reply:
I think that in the ideal world, the autobuilder or whatever CI/CD process is utilized can bear the responsibility of uploading build artifacts into Atlas - which is a much cleaner abstraction anyways, since then it can hold responsibility over any tangential upsert logic and the parser can focus on parsing 😄